plugin.video.itvrs/resources/lib/plugintools.py

```diff
@@ -0,0 +1,497 @@
+# -*- coding: utf-8 -*-
+#---------------------------------------------------------------------------
+# Plugin Tools v1.0.8
+#---------------------------------------------------------------------------
+# License: GPL (http://www.gnu.org/licenses/gpl-3.0.html)
+# Based on code from youtube, parsedom and pelisalacarta addons
+# Author: 
+# Jesús
+# tvalacarta@gmail.com
+# http://www.mimediacenter.info/plugintools
+#---------------------------------------------------------------------------
+# Changelog:
+# 1.0.0
+# - First release
+# 1.0.1
+# - If find_single_match can't find anything, it returns an empty string
+# - Remove addon id from this module, so it remains clean
+# 1.0.2
+# - Added parameter on "add_item" to say that item is playable
+# 1.0.3
+# - Added direct play
+# - Fixed bug when video isPlayable=True
+# 1.0.4
+# - Added get_temp_path, get_runtime_path, get_data_path
+# - Added get_setting, set_setting, open_settings_dialog and get_localized_string
+# - Added keyboard_input
+# - Added message
+# 1.0.5
+# - Added read_body_and_headers for advanced http handling
+# - Added show_picture for picture addons support
+# - Added optional parameters "title" and "hidden" to keyboard_input
+# 1.0.6
+# - Added fanart, show, episode and infolabels to add_item
+# 1.0.7
+# - Added set_view function
+# 1.0.8
+# - Added selector
+#---------------------------------------------------------------------------
+
+import xbmc
+import xbmcplugin
+import xbmcaddon
+import xbmcgui
+
+import urllib
+import urllib2
+import re
+import sys
+import os
+import time
+import socket
+from StringIO import StringIO
+import gzip
+
+module_log_enabled = False
+http_debug_log_enabled = False
+
+LIST = "list"
+THUMBNAIL = "thumbnail"
+MOVIES = "movies"
+TV_SHOWS = "tvshows"
+SEASONS = "seasons"
+EPISODES = "episodes"
+OTHER = "other"
+
+# Write something on XBMC log
+def log(message):
+    xbmc.log(message)
+
+# Write this module messages on XBMC log
+def _log(message):
```

Comment:
please drop this. It will only make parsing the log a confusing task. The addon that is running is `plugin.video.itvrs` and not something called `plugintools`